stdlib/LinearAlgebra/src/tridiag.jl

```diff
@@ -693,6 +697,22 @@ end
     end
 end
 
+Base._reverse(A::Tridiagonal, dims) = reverse!(Matrix(A); dims)
+Base._reverse(A::Tridiagonal, dims::Colon) = Tridiagonal(reverse(A.du), reverse(A.d), reverse(A.dl))
+function Base._reverse!(A::Tridiagonal, dims::Colon)
+    n = length(A.du) # == length(A.dl), & always 1-based
+    # reverse and swap A.dl and A.du:
+    for i in 1:(n >> 1)
+        @inbounds A.dl[i], A.du[i], A.dl[n+1-i], A.du[n+1-i] = A.du[n+1-i], A.dl[n+1-i], A.du[i], A.dl[i]
+    end
+    if isodd(n) # swap middle element
+        i = (n >> 1) + 1
+        @inbounds A.dl[i], A.du[i] = A.du[i], A.dl[i]
+    end
```

Comment:
```suggestion
    @inbounds for i in 1:n
        A.dl[i], A.du[n+1-i] = A.du[n+1-i], A.dl[i]
    end
```
IIUC this is equivalent